belvo/resources/base.py

```diff
@@ -13,9 +13,9 @@ def __init__(self, session: APISession) -> None:
     def session(self) -> APISession:
         return self._session
 
-    def list(self, **kwargs) -> Generator:
+    def list(self, timeout: int = 5, **kwargs) -> Generator:
```

Comment:
:lipstick: For preventing `timeout` to be provided as positional argument (e.g. force `client.list(timeout=10)` instead of `client.list(10)`):
```suggestion
    def list(self, *, timeout: int = 5, **kwargs) -> Generator:
```